spec.emu

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="stylesheet" href="./spec.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/github.min.css">
+<script src="./spec.js"></script>
+<pre class="metadata">
+title: String.prototype.trimStart/trimEnd (plus trimLeft/trimRight)
+stage: 1
+contributors: Sebastian Markbåge, Tom Schuster
+</pre>
+
+<emu-clause id="String.prototype.trim">
+    <h1>String.prototype.trim ( )</h1>
+    <p>The following steps are taken:</p>
+    <emu-alg>
+      1. Let _S_ be *this* value.
+      1. Return ? TrimString(_S_, "start+end").
```

Comment:
i think the strings should be surrounded with `*` so they show up bold (to indicate they're ECMAScript values)